Engine/ac/characterextras.h

```diff
@@ -43,8 +44,9 @@ struct CharacterExtras {
     char  process_idle_this_time;
     char  slow_move_counter;
     short animwait;
+    int blend_mode;
```

Comment:
Since BlendMode is a enum, could you use that enum here instead of type "int"? And everywhere where it's passed as a function parameter.